documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,31 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Example
```

Comment:
What happens if the target already exists? How does a continuous process prevent to re-create a key?

Reply:
the same document template is used in all other CPI commands which was merged previously as suggested by Marcus/Oliver

Reply:
Missing documentation hinders the usage as well as the review. As a consumer, I would like to get as much information as possible. To reuse a template is fine but does not prevent to add step specific information.

Reply:
fixed